packages/mobile/src/harmony-native/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,55 @@
+import type { AvatarProps as HarmonyAvatarProps } from '@audius/harmony'
+import styled from '@emotion/native'
+import type { ImageSourcePropType } from 'react-native/types'
+
+export type AvatarProps = Omit<HarmonyAvatarProps, 'src'> & {
+  src: ImageSourcePropType
```

Comment:
Can we add `| undefined` for the scenarios when we want to show an icon instead of an image (sign up flow, user not signed in)
```suggestion
  src: ImageSourcePropType | undefined
```